### ISSUE TITLE
CLI for plugin

### ISSUE THREAD

**Issue by rootwarp** [dsrvlabs/vatz#316]
## Checklist

- [X] New Feature for the Service/Plugin (Vatz)
- [ ] Enhancement (Vatz)
- [ ] others(etc. e.g, documentation, project policy management)

---
### **Is your feature request related to a problem? Please describe.**
> A clear and concise description of what the problem is. such as Ex. I'm always frustrated

### **Describe the solution you'd like| Ex**
> A clear and concise description of what you want to happen

### **Describe alternatives you've considered**
> A clear and concise description of any alternative solutions or features you have considered.

### **Additional context or comment**
> Add any other context or screenshots about the feature request here.

**Comment by gnongs** [comment]
@rootwarp I think we need to synchronize plugin information with DB and yaml file

How do you think about that ?

**Comment by rootwarp** [comment]
> @rootwarp I think we need to synchronize plugin information with DB and yaml file
> 
> 
> 
> How do you think about that ?

@gnongs Agree. As you know plugin information is managed by two part, yaml and DB so it will make confusion to users.

But now, we have no detail plans regarding to config yaml file. So we have to estimate plan precisely to keep backward compatibility.

It could be good to make this issue public and discuss together.

@xellos00 How do you think?

**Comment by xellos00** [comment]
> > @rootwarp I think we need to synchronize plugin information with DB and yaml file
> > How do you think about that ?
> 
> @gnongs Agree. As you know plugin information is managed by two part, yaml and DB so it will make confusion to users.
> 
> But now, we have no detail plans regarding to config yaml file. So we have to estimate plan precisely to keep backward compatibility.
> 
> It could be good to make this issue public and discuss together.
> 
> @xellos00 How do you think?

@rootwarp @gnongs 
Yes, I agree. It really good idea. 
I prefers discuss it after  we finishes node exporter and version 1.0

**Comment by rootwarp** [comment]
@xellos00 Maybe `Support DB migration if the DB schema is changed` item might be related what you said.

**Comment by xellos00** [comment]
> t DB migration if the DB schema is

yes, it does. let me create a issue, and make relation with this issue to update.
